nomad/grapl-core.nomad

```diff
@@ -258,7 +263,7 @@ job "grapl-core" {
         RUST_BACKTRACE = local.rust_backtrace
         RUST_LOG       = var.rust_log
 
-        OTEL_RESOURCE_ATTRIBUTES = local.default_otel_resource_attributes
+        OTEL_RESOURCE_ATTRIBUTES = "${local.default_otel_resource_attributes},service.version=${var.container_versions["analyzer-dispatcher"]}"
```

Comment:
Surprisingly the `"${var.foo["bar"}"` is the correct syntax. Single quotes are illegal in hcl2 and escaping quotes within interpolated values is not allowed